docker/.env

```diff
@@ -49,7 +50,7 @@ TEST_DATA_URL=https://pds-gamma.jpl.nasa.gov/data/pds4/test-data/registry/urn-na
 HARVEST_DATA_DIR=./test-data/registry-harvest-data
 
 # Elasticsearch URL (the host name is the Elasticsearch service name specified in the docker compose)
-ES_URL=http://elasticsearch:9200
+ES_URL=https://elasticsearch:9200
```

Comment:
Why `https`?

Isn't this communication strictly internal to the host (within Docker)? This extra SSL would just increase CPU costs 🤔 

Is this a multi-host app? I thought it was all launched from a single `docker-compose.yaml`. I don't get it 🤷‍♀️

Reply:
@nutjob4life, I agree with you about the increased CPU costs.

The OpenSearch by default operates with security enabled. It is possible to disable security and run OpenSearch with http.  However, the purpose of this change was to make sure all our components can communicate with a secured OpenSearch (with https and username/password).

I would like to get opinions of @tloubrieu-jpl and @jordanpadams also on the point made by @nutjob4life .

Reply:
Testing https is necessary since the SAs will insist that the AWS OpenSearch service employ it.

Reply:
Oh so we will be using AWS-managed OpenSearch eventually … and this demonstrates we can use `https` to communicate with it.

Thanks for the explanation.